### ISSUE TITLE
colnames not showing in plotScoreHeatmap

### ISSUE THREAD

**Issue by robertamezquita** [LTLA/SingleR#57]
Despite setting plotScoreHeatmap(..., show_colnames = TRUE) to try and pass the pheatmap argument, currently does not allow the override to occur. Makes interpreting the scores heatmap difficult without the colnames (although understandable why you'd want it off for single-cell based score heatmaps). To fix this, had to go into the function and manually set show_colnames to TRUE. 

```
## hotfix to make colnames appear
plotScoreHeatmap <- function(...) {
...
args <- list(mat = scores[, order, drop = FALSE], border_color = NA,
    show_colnames = TRUE, clustering_method = "ward.D2",
    cluster_cols = cluster_cols, breaks = breaks, ...)
...
}
```

**Comment by dtm2451** [comment]
I can certainly see why you might want to do this when subsetting the heatmap to a point when individual names would be visible.

We can recode the typical colnames hiding to work via `if (is.null(args$show_colnames)) { args$show_colnames <- FALSE }.  Then plotScoreHeatmap(..., show_colnames = TRUE) should work naturally.

I'll make the PR shortly.

**Comment by LTLA** [comment]
Why can't we just have a `show_colnames=FALSE` default that gets set in `args`?

Also, make sure your PR is done on `postrelease` so I can merge it into both `master` and `RELEASE_3_10`.

**Comment by robertamezquita** [comment]
Yeah @dtm2451 forgot to mention, this is specifically for the case when classifying by clusters in lieu of single cells! Your fix looks 👍

**Comment by dtm2451** [comment]
That'd work too, @LTLA.  I'd wanted to keep the inputs limited in order to keep the visualization functions from appearing more daunting to learn than they actually are... but it's just one more.

**Comment by LTLA** [comment]
Fixed in 1.0.1 and 1.1.1.
